src/feeder/et_feeder_node.h

```diff
@@ -40,11 +40,6 @@ class ETFeederNode {
   uint32_t comm_tag() const;
 
  private:
-  void assign_attr_val(
-      std::shared_ptr<ChakraProtoMsg::Node> node,
-      int i,
-      void* member);
-
```

Comment:
Removed it because currently we just declare this function without any implementation of it. Previously it is used to read attribute value but for now we turn into a strong-type solution so we do not need it any more. 

As no implementation, keeping it in header might be confusion, so just removed it.